package.json

```diff
@@ -132,7 +132,7 @@
     "pull-stream-to-stream": "^1.3.3",
     "pull-zip": "^2.0.1",
     "read-pkg-up": "^2.0.0",
-    "readable-stream": "1.1.14",
+    "readable-stream": "2.2.2",
```

Comment:
all tests are passing with `readable-stream` 2.2.2. @dignifiedquire  do you remember where is the diagnosis of the issue we found a while back?